.gitignore

```diff
@@ -9,3 +9,4 @@ resources.py
 olive.log
 olive.spec
 test.json
+nbproject/
```

Comment:
Please remove the IDE artifacts from this pull request

Reply:
accept

Reply:
solved